dotnet/src/Functions/Functions.UnitTests/Yaml/PromptExecutionSettingsTypeConverterTests.cs

```diff
@@ -0,0 +1,356 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel;
+using Xunit;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+
+namespace SemanticKernel.Functions.UnitTests.Yaml;
+
+/// <summary>
+/// Tests for <see cref="PromptExecutionSettingsTypeConverter"/>.
+/// </summary>
+public sealed class PromptExecutionSettingsTypeConverterTests
+{
+    private readonly IDeserializer _deserializer;
+
+    private readonly Kernel _kernel;
```

Comment:
nit. (Im very used to see private fields in the beggining)

Some privates methods in the end, some fields in the beggining, suggest moving those to the end